pybossa/cache/site_stats.py

```diff
@@ -421,3 +421,53 @@ def management_dashboard_stats_cached():
         # reset stats for any missing stat
         map(delete_cache_group, management_dashboard_stats)
     return stats_cached
+
+def total_tasks(interval):
+    """
+    Total tasks
+    """
+    sql = text(f'''
+        SELECT count(task.id) as num_tasks
+        FROM task WHERE
+        to_timestamp(task.created, 'YYYY-MM-DD"T"HH24:MI:SS.US') > current_timestamp - interval '{interval}';
+    ''')
+    rows = session.execute(sql).fetchall()
+    return rows[0][0]
+
+def total_taskruns(interval):
+    """
+    Total taskruns
+    """
+    sql = text(f'''
+        SELECT count(task_run.id) as num_taskruns
+        FROM task_run WHERE
+        to_timestamp(task_run.finish_time, 'YYYY-MM-DD"T"HH24:MI:SS.US') > current_timestamp - interval '{interval}';
+    ''')
+    rows = session.execute(sql).fetchall()
+    return rows[0][0]
+
+def total_active_projects(interval):
```

Comment:
If a "active" project is based on "updated" (which makes sense to me), I think we could modify the implementation of ```number_of_created_jobs```.